uk_geo_utils/management/commands/import_onspd.py

```diff
@@ -59,24 +88,23 @@ def import_onspd(self, table_name):
                 "No CSV files found in %s" % (self.data_path)
             )
 
-        cursor = connection.cursor()
-
         self.stdout.write("importing from files..")
         for f in files:
-            self.stdout.write(f)
+            header = self.check_header(f)
+            self.stdout.write(f"Importing {f}")
             with open(f, "r") as fp:
-                cursor.copy_expert(
+                self.cursor.copy_expert(
                     """
                     COPY %s (
                     %s
-                    ) FROM STDIN (FORMAT CSV, DELIMITER ',', quote '"', HEADER);
+                    ) FROM STDIN (FORMAT CSV, DELIMITER ',', quote '"', HEADER MATCH);
```

Comment:
TIL `MATCH` :mortar_board:
There might be one or two other places we are doing a `COPY FROM` where we could usefully add this check.

Reply:
https://github.com/DemocracyClub/UK-Polling-Stations/issues/8151